aws/resources/security_hub.go

```diff
@@ -114,33 +113,33 @@ func (sh *SecurityHub) nukeAll(securityHubArns []string) error {
 
 	// Check for an administrator account
 	// Security hub cannot be disabled with an active administrator account
-	adminAccount, err := sh.Client.GetAdministratorAccountWithContext(sh.Context, &securityhub.GetAdministratorAccountInput{})
+	adminAccount, err := sh.Client.GetAdministratorAccount(sh.Context, &securityhub.GetAdministratorAccountInput{})
 	if err != nil {
 		logging.Errorf("[Failed] Failed to check for administrator account")
 	}
 
 	// Disassociate administrator account if it exists
 	if adminAccount.Administrator != nil {
```

Comment:
hm, `adminAccount` can be null? looks like after `err` handling execution continue with checking `adminAccount` fields